### PR TITLE
feat: added badge on button

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added badges on the button. It is oriented to:

>Left-Top
Right-Top

badgeOnButtons

### CODE REVIEW

**Review by bhanuprakashj — COMMENTED** [review]
code looks good to me!!
one suggestion, instead of having negative top/left/right values try transform translating with negative percentages

File: components/badge-on-button.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    min-width: var(--size-12);
+    padding: 4px;
+    position: absolute;
+    top: -9.6px;
+}
+.left-buttonBadge{
+    left: -9.6px;
+}
+.right-buttonBadge{
+    right: -9.6px;
```
Comment (Hamdrive) [inline components/badge-on-button.css]: consider using `transition: transform()` property to position badge

**Review by Hamdrive — COMMENTED** [review]
Code looks clean, have suggested changes for transform property

File: components/badge-on-button.css (hunk 1 above)
Comment (GuptaEesh) [reply components/badge-on-button.css]: Done✅

**Review by Hamdrive — APPROVED** [review]
Good work done using `transform` property. All looks good and can be merged!

**Pull request merged**